api/nodes/urls.py

```diff
@@ -7,6 +7,7 @@
     # url(r'^blog/', include('blog.urls')),
     url(r'^$', views.NodeList.as_view(), name='node-list'),
     url(r'^(?P<pk>\w+)/$', views.NodeDetail.as_view(), name='node-detail'),
+    url(r'^(?P<pk>\w+)/confirm/(?P<token>\w+)$', views.NodeDeleteConfirm.as_view(), name='node-delete-confirm'),
```

Comment:
The majority of rules in this file end with a trailing slash. Is there any reason this one shouldn't?